### ISSUE TITLE
Table and Column Descriptions: two types: captured and enriched

### ISSUE THREAD

**Issue by DovileKr** [open-metadata/OpenMetadata#10326]
**Is your feature request related to a problem? Please describe.**
When scanning database metadata, we capture table and column descriptions as taken from the application. However, we also want to enrich it will additional commentary, clarification of business process and so on. Currently there is no way to tell if description came from application or if it was enriched by OMD users. 

**Describe the solution you'd like**
I would like to have two description types. One which is ingested and one that can be enriched by community. Meaning:
-  auto, populated by ingestion service from the source metadata. It will often be blank or short, so no need for much space); if it is blank, it should be collapsed,  ideally. 
-  and one for community, additional updates, which is not touch by re-ingesting the metadata.

**Describe alternatives you've considered**
Using comments. But comments are difficult to consolidate and good metadata might be lost in the comments thread.

**Additional context**
Inspiration from another data catalog tool
![image](https://user-images.githubusercontent.com/26750060/221185667-ff301c1c-034b-4a89-a511-124479afc058.png)
https://docs.castordoc.com/assets/tables#columns

**Comment by chirag-madlani** [comment]
@harshach @pmbrull Do we have any flag to identity above?

**Comment by pmbrull** [comment]
@DovileKr Not sure I follow the goal of this. How I see descriptions is as a way to centralise all known information from an asset, and make sure that we keep it up-to-date.

You should decide if OM becomes the source of truth, or other tools such as dbt. This is why by default, we won't overwrite the description once it reaches OM.

What would happen if the database description gets outdated and Data Stewards add the information in another "description"? How would users know which one to use?

I believe this might carry out more misalignment than actual value. I'll be closing this for now. Let us know your thoughts.

Thanks

**Comment by DovileKr** [comment]
@pmbrull thank you for the message. I want to share the idea why this idea came across.

in BQ we have a table for sales.

- Tables has description as per our standards no table should be created without description.  Metadata was ingested to OMD
- A couple of months later, a new source was added to the table and also the update logic has changed. Table was recreated (same in BQ). So table description has changed. Also description for an existing column as logic changed slightly.

Data engineer has changed table and column description in BQ. And now it is not in sync anymore with OMD. Tables aren't created using something like dbt, but SQL queries...

Here I can
- ask DE to update both BQ and OMD 
- have DE update in OMD and have BQ read descriptions from OMD
- or simply ignore the difference

So in summary yes - small inconvenience.  Maybe more prominent for something like SaaS vendors. Cause if they change some meta like descriptions, will be hard to notice.
